src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1

```diff
@@ -4207,4 +4207,38 @@ class Project: ADOSVTBase
         }
         return $controlResult;
     }
+
+    hidden [ControlResult] CheckDisabledCreationOfClassicPipeline([ControlResult] $controlResult)
+    {
+        $controlResult.VerificationResult = [VerificationResult]::Failed
+        if($this.PipelineSettingsObj)
+        {
+            $orgLevelScope = $this.PipelineSettingsObj.disableClassicPipelineCreation.orgEnabled;
+            $prjLevelScope = $this.PipelineSettingsObj.disableClassicPipelineCreation.enabled;
+            $controlResult.AdditionalInfoInCSV ="NA"
+
+            if($prjLevelScope -eq $true )
+            {
+                $controlResult.AddMessage([VerificationResult]::Passed, "Creation of classic build and classic release pipelines is disabled.");
+            }
+            else
+            {
+                $controlResult.AddMessage([VerificationResult]::Failed, "Creation of classic build and classic release pipelines is not disabled.");
+            }
+
+            if($orgLevelScope -eq $true )
```

Comment:
If else are not written correctly. If prjLevelScope is true our control should pass. But if code is written in this manner, VerificationResult will always come in accordance of orgLevelScope. 

Case prjLevelScope= true and orgLevelScope=false.
VerificationResult will be Failed even though it should be passed

Reply:
While I was debugging the code for this control both the messages were seen. That is the project level result and org level result.
Will check it again.

Reply:
Both messages will be seen but verificationResult populated will be wrong

Reply:
okay, removed the org level check.